File: NoroffAlumni/src/main/java/no/experisAcadmey/trondheim/NoroffAlumni/models/DTOs/eventDTOs/EventDTO.java
package no.experisAcadmey.trondheim.NoroffAlumni.models.DTOs.eventDTOs;

import lombok.Data;
import no.experisAcadmey.trondheim.NoroffAlumni.models.User;

import java.util.Set;

@Data
public class EventDTO {
    private Long id;
    private String createdBy;
    private String groupId;
    private String title;
    private String description;
    private String date;
    private String time;
    private String location;
    private String name;
    private Set<String> participants;
    private int theme;
}

File: noroff-alumni-frontend/src/models/EventDTO.ts
type EventDTO = {
  id: number;
  createdBy: string;
  groupId: string;
  title: string;
  description: string;
  date: string;
  time: string;
  location: string;
  name: string;
  participants: string[];
  theme: number;
}

export default EventDTO;